tests/old/1_test_encryption.py

```diff
@@ -0,0 +1,81 @@
+import base64
```

Comment:
There should be no old and new tests. Please fix naming. Also test names should start with ``test__`` to enable discovery.